client/js/components/procedure/SegmentsList/FilterFlyout.vue

```diff
@@ -247,7 +249,7 @@ export default {
           condition: {
             path: this.path,
             value: id,
-            operator: this.operator
+            operator: id === "unassigned"? "IS NULL" : this.operator
```

Comment:
```suggestion
            operator: id === 'unassigned' ? 'IS NULL' : this.operator
```
Please use single quotes instead of double ones.